sources/csv/README.md

```diff
@@ -0,0 +1,114 @@
+      ]
+    },
+    {
+      "table_name": "Singers",
+      "file_patterns": [
+        "/Users/username/Desktop/harbourbridge/Singers_1.csv"
+        "/Users/username/Desktop/harbourbridge/Singers_2.csv"
+        "/Users/username/Desktop/harbourbridge/Singers_3.csv"
+      ],
+      "columns": [
+        {"column_name": "SingerId", "type_name": "INT64"},
+        {"column_name": "FirstName", "type_name": "STRING(100)"},
+        {"column_name": "LastName", "type_name": "STRING"}
+      ]
+    }
+]
+```
+**CAVEATS:**
+- File patterns do not accept regular expressions. Provide the path inside 
+double quotes.
+- The type_name should be identical to the types in spanner schema. Only 
+for `STRING` and `BYTES`, the length can be optionally omitted.
+
+### CSV File Format
+- Each column in a row should be separated by a `,`(comma).
+- The first row in each file should contain the column names, which should
+be same as the corresponding Spanner column names.
+- Remove trailing spaces, tabs in the column name headers.
+
+**Sample CSV:**
+```
+bool_col,byte_col,date_col,float_col,int_col,numeric_col,string_col,timestamp_col,json_col
+true,bytevalue,2020-12-09,15.13,100,39.94,Helloworld,2019-10-29 05:30:00,"{""key1"": ""value1"", ""key2"": ""value2""}"
+```
+**CSV Data Type Considerations:**
+- We only support scalar data types right now. Sample data format for each type
+provided in the snippet above.
+- The only supported data format right now is **RFC3339 full-date format**.
+- The only supported timestamp format right now is **ISO 8601**.
+- The format to escape the quotes in json is adding an additional `"` in front
+of the double quote. `\` does not work. Also enclose the whole data inside "".
+Some modification might be required since most databases do not export CSVs with escaping quotes like mentioned.
+
+
+## Example CSV Usage
+
+The following examples assume a `harbourbridge` alias has been setup as 
+described in the [Installing HarbourBridge](https://github.com/cloudspannerecosystem/harbourbridge#installing-harbourbridge) section of the main README.
+
+
+To run harbourbridge for loading CSVs, run the following command
+
+```sh
+harbourbridge data -source=csv -source-profile="manifest=manifest_file.json" -target-profile="instance=my-instance,dbname=my-db" 
+```
+
+You can also specify the project in the target profile optionally. If not
+provided, HarbourBridge will search for the project ID in the gcloud 
+config, followed by the environment variable `GCLOUD_PROJECT`. Project, Instance and DBName
+fields in the target profile are mandatory to identify which Spanner database
+to write to.
+
+## Foreign Keys and Interleaved Tables
+
+HarbourBridge currently does not guarantee data conversion if the Spanner
+schema has Foreign Keys or Interleaved Tables in the final schema.
+It is recommended to create Foreign Keys only after data conversion
+has been completed.
+
+For interleaved tables, it may not be possible to change the schema later.
```

Comment:
Does the manifest provide an option to give the order in which tables should be imported ? If it does, we should use that here, else we might see a lot of migrations failing.

Reply:
The order in which the manifest lists is the same order in which it will be imported. Updated the documentation to make it clearer.